sui/types-utils.js

```diff
@@ -46,6 +52,11 @@ const proofStruct = bcs.struct('Proof', {
     signatures: bcs.vector(bcs.vector(bcs.u8())),
 });
 
+const gasServiceStruct = bcs.struct('GasService', {
```

Comment:
add this to cgp sui PR as well?

Reply:
should we move the whole file to `axelar-cgp-sui`?

Reply:
we already have a bunch of these defined there, need to add the missing ones